pkg/sentry/fsimpl/fuse/fusefs.go

```diff
@@ -490,7 +509,8 @@ func (i *inode) Open(ctx context.Context, rp *vfs.ResolvingPath, d *kernfs.Dentr
 // Lookup implements kernfs.Inode.Lookup.
 func (i *inode) Lookup(ctx context.Context, name string) (kernfs.Inode, error) {
 	in := linux.FUSELookupIn{Name: linux.CString(name)}
-	return i.newEntry(ctx, name, 0, linux.FUSE_LOOKUP, &in)
+	node, err := i.newEntry(ctx, name, 0, linux.FUSE_LOOKUP, &in)
```

Comment:
Are you sure that decomposition is needed? Why not leave it as it was? Same for all the later cases.

Reply:
I added it to support fetching2 of 3 results, but forgot to remove after later change. Fixed now.